src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -189,8 +189,7 @@ namespace BaseboardElectric {
         bool ErrorsFound(false); // If errors detected in input
 
         auto &baseboard = state.dataBaseboardElectric;
-        auto &cCurrentModuleObject = state.dataIPShortCut->cCurrentModuleObject;
-        cCurrentModuleObject = cCMO_BBRadiator_Electric;
+        std::string cCurrentModuleObject = cCMO_BBRadiator_Electric;
```

Comment:
This should be a `std::string_view`. Using `std::string` makes a deep copy here, which you don't need.